TaskWithSmartPointers/MySharedPtr.h

```diff
@@ -0,0 +1,128 @@
+#pragma once
+#include <iostream>
+// #include <atomic>
+#include <mutex> 
+
+template <class T>
+class MySharedPtr
+{
+public:
+	std::mutex mtx;
```

Comment:
Move to other variables under the private.

Reply:
Done